webapp/src/Service/ConfigurationService.php

```diff
@@ -70,6 +70,12 @@ public function get(string $name, bool $onlyIfPublic = false)
         return $value;
     }
 
+    public function getCategory(string $name): ?string
+    {
+        $spec = $this->getConfigSpecification()[$name] ?? null;
```

Comment:
Which configs don't have a category? And why should they not have a category?

Reply:
Done.